docs/content/dagster-cloud/getting-started/getting-started-with-serverless-deployment.mdx

```diff
@@ -0,0 +1,137 @@
+
+You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a Dagster Cloud organization. When your organization is created, a single deployment, named `prod`, will also be created. Deployments are standalone environments, allowing you to operate independent instances of Dagster with separately managed permissions.
+
+---
+
+## Step 2: Select Serverless as your deployment type
+
+Serverless deployment is a fully-managed version of Dagster Cloud that allows you to run your Dagster jobs without spinning up any infrastructure. While Serverless is the easiest way to get started with Dagster, this deployment type works best with jobs that aren't computationally expensive. Refer to the [Serverless guide](/dagster-cloud/deployment/serverless) for more info.
+
+If you find that Serverless isn't right for you, you can switch to a Hybrid deployment at any time.
+
+**To use this deployment type**, click **Select** under **Serverless deployment** and confirm when prompted.
+
+---
+
+## Step 3: Deploy your code
+
+Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
+
+A code location specifies a single Python package or file that defines your Dagster code. Each Dagster Cloud deployment can include code from one or more code locations. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
+
+<details>
+	<summary><strong>USING GITHUB</strong></summary>
+
+1. Click **Choose account or organization** and sign into your GitHub account.
+2. Select the account or organization where Dagster should be installed.
+3. After the installation and authorization is complete, you'll be redirected back to Dagster Cloud.
+4. After Dagster Cloud is connected, you'll be prompted to either:
+
+   - **Clone the Dagster Cloud starter project** - This option clones the [dagster-cloud-template](https://github.com/dagster-io/dagster-cloud-template) repository.
+
+     1. Enter a **Repository name**.
+     2. Select a **Repository visibility**.
+     3. Click **Clone and deploy.**
+
+   - **Use an existing repo** - This option uses an existing repository in your GitHub account.
+     1. Select a **repository**.
+     2. Click **Deploy**.
+
+</details>
+
+<details>
+	<summary><strong>USING THE DAGSTER-CLOUD CLI</strong></summary>
+
+To complete this step using the CLI, you can use your own Dagster code or the [`dagster-cloud-examples` code](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). Our examples use the `dagster-cloud-examples` package.
```

Comment:
I know Ben mentioned that having Docker installed was also a requirement, but I wasn't sure if there was anything else around that we need to mention. If it's just as simple as "You need to have Docker installed", I'll add that in.

Reply:
I think it's just docker